tensorboard/data/server_ingester.py

```diff
@@ -70,9 +71,15 @@ def start(self):
         tmpdir = tempfile.TemporaryDirectory(prefix="tensorboard_data_server_")
         port_file_path = os.path.join(tmpdir.name, "port")
 
+        if self._reload_interval <= 0:
```

Comment:
For `reload_interval` values < 0, based on [1], I think we used to avoid loading/ingesting completely, which differs from `reload_interval` 0.  Should we preserve this old behavior?

[1] https://github.com/tensorflow/tensorboard/blob/3c1503df6a6c2130a96c60804007065bcccb6537/tensorboard/backend/event_processing/data_ingester.py#L92

Reply:
Indeed (and it’s even documented). I considered it, but it didn’t seem
very useful to me to teach the data server to admit a mode that does not
serve data. The history shows that this was added in #1001 to accomodate
DB mode, but that kind of accommodation is now performed by the
ingestion system.

Maybe in a follow-up PR I can revert the special treatment for negative
reload intervals altogether, now that DB mode is gone.

Reply:
Sent #4583.

Reply:
Thanks for referencing the original commit.  If we think it's a vestige of DB mode and has no other real utility, dropping support sounds good to me.

If we are dropping the negative interval logic in this PR, can we either add a TODO or file an issue on GH to track removing it in the other places?  I think it's also probably worth mentioning in the next release notes as well.

Reply:
Wow that was fast :)

Since the followup PR is out, a TODO/issue isn't really needed.

Reply:
Yep. Merged.